openapi.yml

```diff
@@ -363,7 +363,23 @@ components:
       type: string
       pattern: '^245[0-9]{8}$'
       example: '24503259768'
-    CatalogLink:
+    CreatedInFolioIdentifier:
+      description: A record identifier created in Folio
+      type: string
+      pattern: '^in\d+$'
+      example: 'in11403803'
+    MigratedFromSymphonyIdentifier:
+      description: A record identifier migrated from Symphony
+      type: string
+      pattern: '^a\d+$'
+      example: 'a11403803'
```

Comment:
It looks like existing Symphony identifiers can be `123:456`. Does this need to allow?

Reply:
@justinlittman I'm still learning about Symphony identifiers--could you point me to an example of one of those with the `:` in it?

Reply:
@arcadiafalcone do you have thoughts on ☝🏻 ?